src/formpack/reporting/export.py

```
@@ -526,3 +528,83 @@ def __get_extra_mapping_values(self, section):
+                            'label': 'I am question 2, unconstrained by
+                                     'predetermined values!'
+                        }
+                    }
+                '''
+                question_dict = OrderedDict()
+                for field in all_fields:
+                    # Even with `multiple_select='summary'`, we can still get
+                    # multiple names and labels per question for things like
+                    # `FormGPSField` (`geopoint`)
+                    xml_names = field.get_labels(lang=UNSPECIFIED_TRANSLATION,
+                                                 multiple_select='summary')
+                    assert xml_names[0] == field.name
+                    labels = field.get_labels(lang=translation,
+                                              multiple_select='summary')
+                    for name, label in zip(xml_names, labels):
+                        question_dict[name] = {
+                            'label': label,
+                            'data_type': field.data_type,
+                        }
+                    if hasattr(field, 'choice'):
+                        choices = OrderedDict()
+                        for option in field.choice.options.keys():
+                            choices[option] = field.get_translation(
+                                val=option, lang=translation
+                            )
+                        question_dict[field.name]['values'] = choices
+                # Convert the question/choice names and labels into SPSS
+                # commands
+                spss_label_commands = spss_labels_from_variables_dict(
+                    question_dict)
+                # Write the SPSS commands into a file for this particular
+                # language
+                title = self.formpack.title
+                rest_of_filename = ' - '.join(
+                    ('', translation, 'SPSS labels.sps')
```

We may want to assert `translation` is not None otherwise this line raises an Exception. 
Maybe we could raise the Exception at the beginning of the method for forms without translations.